index.bs

```diff
@@ -1542,17 +1592,9 @@ This is a [=static command=].
           browserName: text,
           browserVersion: text,
           platformName: text,
-          proxy: {
-            ? proxyType: "pac" / "direct" / "autodetect" / "system" / "manual",
-            ? proxyAutoconfigUrl: text,
-            ? ftpProxy: text,
-            ? httpProxy: text,
-            ? noProxy: [*text],
-            ? sslProxy: text,
-            ? socksProxy: text,
-            ? socksVersion: 0..255,
-          },
           setWindowRect: bool,
+          ? proxy: session.ProxyConfiguration,
+          ? webSocketUrl: bool,
```

Comment:
The boolean is saying "we return support web socket". Naming is kinda poor, we could have just said "webSocket". See the `webSocketUrl capability serialization algorithm`.

Reply:
But in WebDriver classic this is not a boolean for the new session response. So isn't that a conflict?

Reply:
Can you point to the place in the spec where it says this? I don't believe `webSocketUrl` exists in WebDriver classic.

Reply:
Well, it doesn't exist in the spec but is used for the BiDi (WebSocket) opt-in. See the steps 6 and 7 of https://w3c.github.io/webdriver-bidi/#establishing.

Reply:
Sorry, @whimboo is correct that this is a bug, and the return type is a string (for HTTP+BiDi sessions; it's a boolean for BiDi-only sessions). Apologies for not catching that. It's also true that the spec is confusing.

The key algorithm is https://w3c.github.io/webdriver/#ref-for-dfn-remote-end-steps-6

In step 4 we process capabilities and (assuming there's a match and we're starting a websocket connection), end up with an object that contains `{webSocketUrl: true}`.

In step 9 we call into https://w3c.github.io/webdriver-bidi/#establishing which _updates_ the `capabilities` object so that it contains the actual URL as the value of the `webSocketUrl` property.

Reply:
Oh my, indeed that is very confusing. Here is the fix: https://github.com/w3c/webdriver-bidi/pull/606